packages/now-cli/test/integration.js

```diff
@@ -500,22 +495,67 @@ test('scale the alias', async t => {
     binaryPath,
     ['scale', context.alias, 'bru', '1', ...defaultArgs],
     {
-      reject: false
+      reject: false,
     }
   );
 
   t.is(code, 0, `Received:\n${stdout}\n${stderr}`);
   t.true(stdout.includes(`(min: 1, max: 1)`));
 });
 
-test('remove the alias', async t => {
+test('remove the explicit alias', async t => {
   const goal = `> Success! Alias ${context.alias} removed`;
 
   const { stdout, code } = await execa(
     binaryPath,
     ['alias', 'rm', context.alias, '--yes', ...defaultArgs],
     {
-      reject: false
+      reject: false,
+    }
+  );
+
+  t.is(code, 0);
+  t.true(stdout.startsWith(goal));
+});
+
+test('create an alias from "now.json" `alias` for deployment', async t => {
+  const target = fixture('dockerfile');
+
+  // Add the `alias` field to the "now.json" file
+  const jsonPath = path.join(target, 'now.json');
+  const json = JSON.parse(await readFile(jsonPath, 'utf8'));
+  json.alias = `${session}-from-nowjson.now.sh`;
```

Comment:
Maybe we can use an alias like `${random}.${username}.now.sh`, since the other one could be taken, and then the test would fail.

Reply:
The logic was lifted from the other `now alias` test, but with the "-from-nowjson" portion added. I don't think the collision is very likely, as the `session` is already a randomly generated string.